crypto/x509/x509_lu.c

```diff
@@ -289,24 +289,25 @@ X509_OBJECT *X509_STORE_CTX_get_obj_by_subject(X509_STORE_CTX *vs,
 int X509_STORE_CTX_get_by_subject(X509_STORE_CTX *vs, X509_LOOKUP_TYPE type,
                                   X509_NAME *name, X509_OBJECT *ret)
 {
-    X509_STORE *ctx = vs->ctx;
+    X509_STORE *store = vs->ctx;
```

Comment:
It's to wonder why the X509 store is called `ctx` in `X509_STORE_CTX`...  Considering that's a opaque type, a name change might be appropriate as part of this cleanup.  What do you think?

Reply:
(or maybe you'd prefer that to be for another PR...)

Reply:
Another PR..
I NEVER like to see ctx->ctx  :)

Reply:
I assume just master since its just a cleanup?